WebDriverAgentLib/Utilities/FBElementTypeTransformer.m

```diff
@@ -117,7 +123,11 @@ + (XCUIElementType)elementTypeWithTypeName:(NSString *)typeName
   [self createMapping];
   NSNumber *type = StringToElementTypeMapping[typeName];
   if (!type) {
-    NSString *reason = [NSString stringWithFormat:@"Invalid argument for class used '%@'. Did you mean XCUIElementType%@?", typeName, typeName];
+    if ([typeName hasPrefix:(NSString *)FB_ELEMENT_TYPE_PREFIX] && typeName.length > FB_ELEMENT_TYPE_PREFIX.length) {
+      [FBLogger logFmt:@"Mapping an unknown element type '%@' to XCUIElementTypeOther. Consider reviewing the ElementTypeToStringMapping", typeName];
+      return XCUIElementTypeOther;
+    }
+    NSString *reason = [NSString stringWithFormat:@"Invalid argument for class used '%@'. Did you mean %@%@?", typeName, FB_ELEMENT_TYPE_PREFIX, typeName];
```

Comment:
interesting...
What about adding a message like "Please tell Appium developers from https://github.com/appium/appium/issues/new about the new type"?

Reply:
It should never be the case that a class name start with something which is not `XCUIElementType`